internal/indexer/definitions/ptp.yaml

```diff
@@ -93,31 +64,113 @@ irc:
+            torrentSize: "1899875175"
+            releaseTags: H.264/MKV/WEB/720p
+            editionTags: ""
+            year: "2023"
+            title: "Last Straw"
+            torrentName: Last.Straw.2023.REPACK.720p.AMZN.WEB-DL.DDP5.1.H.264-FLUX
+            tags: thriller,horror
+        - line: 1:0:0:1:357805:1332009:9k6p:tt24249072:1899875175|H.264/MKV/WEB/720p||2023|Last Straw|Last.Straw.2023.REPACK.720p.AMZN.WEB-DL.DDP5.1.H.264-FLUX|thriller,horror
           expect:
-            year: "1972"
-            releaseTags: x264 / Blu-ray / MKV / 1080p / Freeleech!
-            freeleech: Freeleech
-            torrentId: "964303"
-            torrentName: That.Other.Movie.1972.1080p.BluRay.FLAC.x264-GROUP
-            tags: comedy, drama, romance
-        pattern: '.* \[(.*)\] (?:by .*)?\| (.*?(?: \/ (Freeleech)!)?) \| .* \| (.*) \| (.*) \| (.*)'
+            announceType: 1
+            ptpFL: 0
+            scene: 0
+            category: 1
+            groupId: "357805"
+            torrentId: "1332009"
+            key: "9k6p"
+            imdb: "tt24249072"
+            torrentSize: "1899875175"
+            releaseTags: H.264/MKV/WEB/720p
+            editionTags: ""
+            year: "2023"
+            title: "Last Straw"
+            torrentName: Last.Straw.2023.REPACK.720p.AMZN.WEB-DL.DDP5.1.H.264-FLUX
+            tags: thriller,horror
+        - line: 1:0:0:1:21108:1332339:ncfe:tt0107426:63996413804|BD66/m2ts/Blu-ray/2160p|Dolby Atmos/Dolby Vision|1993|Little Buddha|Little.Buddha.1993.2160p.FRA.UHD.Blu-ray.DV.HDR.HEVC.DTS-HD.MA.5.1|drama,italian
+          expect:
+            announceType: 1
+            ptpFL: 0
+            scene: 0
+            category: 1
+            groupId: "21108"
+            torrentId: "1332339"
+            key: "ncfe"
+            imdb: "tt0107426"
+            torrentSize: "63996413804"
+            releaseTags: BD66/m2ts/Blu-ray/2160p
+            editionTags: "Dolby Atmos/Dolby Vision"
+            year: "1993"
+            title: "Little Buddha"
+            torrentName: Little.Buddha.1993.2160p.FRA.UHD.Blu-ray.DV.HDR.HEVC.DTS-HD.MA.5.1
+            tags: drama,italian
+        # <boolnew>:<enum_fl>:<boolscene>:<category>:<gid>:<tid>:<key>:tt<imdbid>:<sizebytes>|<format>/<container>/<source>/<resolution>|<edition_tags>|<year>|<title>|<releasename>|<taglist>
+        pattern: '(\d):(\d):(\d):(\d):(\d+):(\d+):(.+):(tt?\d+):(\d+)\|(.*)\|(.*)\|(\d+)\|(.+)\|(.+)\|(.+)'
         vars:
-          - year
-          - releaseTags
-          - freeleech
+          - announceType
+          - ptpFL
+          - scene
+          - category
+          - groupId
           - torrentId
+          - key
+          - imdb
+          - torrentSize
+          - releaseTags
+          - editionTags
+          - year
+          - title
           - torrentName
           - tags
 
+    mapCustomVars:
+      announceType:
+        "0":
+          announceType: NEW
+        "1":
+          announceType: PROMO
```

Comment:
shouldn't his be the other way around? 🤔

Reply:
Should it? :thinking:

Reply:
at least in the announce it's 1 for new stuff, haven't seen one that is not 1 to compare tho 😄

Reply:
Agree with nuxencs here.

Reply:
It's about the be changed to something like this so the above is correct for now.

0 - new - only on fresh upload
1 - freeleeched
2 - reseeded
3 - marked golden popcorn
4 - checked ... ?